src/storage/src/sst.rs

```diff
@@ -285,13 +295,53 @@ impl FileHandleInner {
     }
 }
 
+#[derive(Debug, Snafu)]
+pub struct ParseIdError {
+    source: uuid::Error,
+}
+
+/// Unique id for [SST File].
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub struct FileId(Uuid);
+
+impl FileId {
+    /// Returns a new unique [FileId] randomly.
+    pub fn random() -> FileId {
```

Comment:
What about renaming it to `uuid`?The name will describe its behaviour: a unique Id for a file. It's much better than `random`.

Reply:
I think this function returns an auto-assigned, random and unique ID. So, maybe `random_uuid` is a better choice? 
 
ps: The `ProcedureId` struct plays a similar role with this `FileId`, and it uses a `random` as a init function. Should we make them consistent?

Reply:
> ps: The ProcedureId struct plays a similar role with this FileId, and it uses a random as a init function. Should we > make them consistent?

If so, let's keep it to be `random`.